index.html

```diff
@@ -0,0 +1,202 @@
+
+      <div class="trainers-section__wrapper wrapper">
+        <div class="trainers-section__trainer">
+          <img src="./assests/img/expert-trainers/image-trainer-1.jpg" alt="Trainer Image">
+          <h3>Sarra Miller</h3>
+          <span>Spinning Trainer</span>
+          <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Distinctio, ad.</p>
+        </div>
+        <div class="trainers-section__trainer">
+          <img src="./assests/img/expert-trainers/image-trainer-1.jpg" alt="Trainer Image">
+          <h3>Sarra Miller</h3>
+          <span>Spinning Trainer</span>
+          <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Distinctio, ad.</p>
+        </div>
+        <div class="trainers-section__trainer">
+          <img src="./assests/img/expert-trainers/image-trainer-1.jpg" alt="Trainer Image">
+          <h3>Sarra Miller</h3>
+          <span>Spinning Trainer</span>
+          <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Distinctio, ad.</p>
+        </div>
+      </div>
+
+    </div>
+  </section>
+
+  <!-- Promotions Section -->
+  <section class="promotions-section">
+    <div class="promotions-section__wrapper">
+      <div class="promotions-section__promotion">
+        <h2>try for free</h2>
+        <h3>free 5 days pass</h3>
+        <button class="btn btn-primary">get in touch</button>
+      </div>
+      <div class="promotions-section__contact">
+        <h2>call us and try</h2>
+        <h3>957 547 4545</h3>
+        <button class="btn btn-primary">get in touch</button>
+      </div>
+    </div>
+  </section>
+
+  <!-- Footer Section -->
+  <footer class="footer-section">
+    <div class="footer-section__wrapper wrapper">
+
+      <div class="footer-section__subscrition">
+        <h2>Fitness</h2>
+        <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Ut ad amet, eaque, distinctio temporibus nihil, animi ullam expedita natus magni dolores! Voluptas ipsa nesciunt laborum ea ducimus repudiandae eius praesentium.</p>
+        <div class="footer-section__social-links">
+          <span>follow us</span>
+          <ul>
+            <li>facebook</li>
+            <li>twitter</li>
+            <li>instagram</li>
+            <li>pinterest</li>
+            <li>google +</li>
+            <li>youtube</li>
+            <li>not reddit</li>
+            <li>tumblr</li>
+          </ul>
+        </div>
+      </div>
+      <div class="footer-section__blogs">
+        Blog Posts
+      </div>
+      <div class="footer-section__feed">
+        instagram or twitter news feed
+      </div>
+      <form class="footer-section__subscribe">
+        <input type="email" name="subscribe" placeholder="Your Email..." id="footer-section__subscribe">
+        <button class="btn btn-primary" type="submit">subscribe</button>
+      </form>
+    </div>
+
+    <div class="footer-section__credits">
+      footer credits
+    </div>
+  </footer>
+
+  <script src="./assests/js/main.js"></script>
```

Comment:
unused file, better to remove it. the less our code base is, the better. ;)

Reply:
done!